main.go

```diff
@@ -191,6 +215,99 @@ func tryRebalance(ctx context.Context, r *regolancer, attempt *int) (err error,
 	return nil, true
 }
 
+func tryRapidRebalance(ctx context.Context, r *regolancer, from, to uint64, route *lnrpc.Route, amt int64) (successfullAtempts int, err error) {
+
+	rapidAttempt := 0
+
+	for {
+
+		log.Printf("Rapid rebalance attempt %s", hiWhiteColor(rapidAttempt))
```

Comment:
Here we need hiWhiteColor(rapidAttempt+1) if we count from zero

Reply:
ok will squash some commits now, getting too much lol

Reply:
Indeed, just one is enough!

Reply:
ok will do